### ISSUE TITLE
Automatically add license references to all source files 

### ISSUE THREAD

**Issue by Mobrockers** [stryker-mutator/stryker-net#625]
I understand  this part as the recommendation to enclose some information within each file. But this is a should, not a must.
>The text should be enclosed in the appropriate
>     comment syntax for the file format. We also recommend that a
>    file or class name and description of purpose be included on the
 >   same "printed page" as the copyright notice for easier
 >  identification within third-party archives.

I found this [thread](https://softwareengineering.stackexchange.com/questions/125836/do-you-have-to-include-a-license-notice-with-every-source-file) on this very topic.
I just pushed a commit without the header. 
I still think you should have some minimal header info, such as project 'StrykerMutator.Net', license Apache 2 and maybe a year

_Originally posted by @dupdob in https://github.com/stryker-mutator/stryker-net/pull/624_

**Comment by Mobrockers** [comment]
@dupdob we have discussed this internally and we have decided that we consider license headers unnecessary since they only serve to protect third-parties that copy the code without considering the license attached to it. I'll reconsider if a third-party requests that we add a license header. Until then I don't see the benefit for us to put in the effort. 

Feel free to add the license header to changes you submit if you do feel it is important. The company name can be stryker-mutator or the project name can be stryker-net.
